zipkin-web/package.json

```diff
@@ -20,7 +20,8 @@
     "flightjs": "^1.5.1",
     "jquery": "^2.2.0",
     "js-cookie": "^2.1.0",
-    "moment": "^2.11.1",
+    "lodash": "^4.5.0",
+    "moment": "^2.11.2",
```

Comment:
great. I know we had an outstanding issue wrt moment.. particularly the duration thing, which defaults to user hard-entering micros, should be easier now! #864 cc @danchia @yurishkuro